src/tree.rs

```diff
@@ -169,6 +172,8 @@ impl<Pane> Tree<Pane> {
     ///
     /// The tree will use upp all the available space - nothing more, nothing less.
     pub fn ui(&mut self, behavior: &mut dyn Behavior<Pane>, ui: &mut Ui) {
+        self.id = ui.id();
+
```

Comment:
It actually wasn't, because if we don't update it every frame the id might not be the correct one. But re-looking at this PR I think it might be neater to remove storing the id entirely and use relative ids everywhere. It will require a argument change to quite a few of the functions then, as it'll require the tree to pass down the "relative" id.